applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -476,14 +472,17 @@ static int change_active_brdcast_src(void)
 	if (++active_stream.brdcast_src_name_idx >= ARRAY_SIZE(brdcast_src_names)) {
 		active_stream.brdcast_src_name_idx = 0;
 	}
+	LOG_INF("Switching to %s", brdcast_src_names[active_stream.brdcast_src_name_idx]);
 
-	LOG_DBG("Switching to %s", brdcast_src_names[active_stream.brdcast_src_name_idx]);
-
+	LOG_DBG("Restarting scanning for broadcast sources");
 	ret = bt_audio_broadcast_sink_scan_start(BT_LE_SCAN_PASSIVE);
-	if (ret) {
-		return ret;
+	if (ret && ret != -EALREADY) {
+		LOG_ERR("Unable to start scanning for broadcast sources");
```

Comment:
I guess it should return the error as well?

Reply:
Yes!